test/common/buffer/buffer_test.cc

```diff
@@ -1075,6 +1076,33 @@ TEST(BufferHelperTest, WriteBEI64) {
     EXPECT_EQ("\x7F\xFF\xFF\xFF\xFF\xFF\xFF\xFF", buffer.toString());
   }
 }
+TEST(BufferHelperTest, AddFragments) {
```

Comment:
nit: can you be consistent with in the expectations order e.g. some calls are `EXPECT_EQ(actual, expected)`, others are `EXPECT_EQ(expected, actual)`.

Reply:
get it.